microcosm_logging/formatters.py

```diff
@@ -1,6 +1,6 @@
 from logging import Formatter
 
-from pythonjsonlogger.jsonlogger import merge_record_extra
+from pythonjsonlogger.core import merge_record_extra
```

Comment:
this was breaking `microcosm-flask` when trying to create a new release. 

at some point (apparently [a couple of years ago](https://nhairs.github.io/python-json-logger/latest/changelog/#deprecated))  this package (`python-json-logger`) moved this module to this new location.

Reply:
cc/ @salmanmashayekh @lfmoisa-globality FYI ^